Documentation/project-docs/developer-guide.md

```diff
@@ -96,67 +112,61 @@ build-managed -debug -buildArch=x64
 build-managed -debug -buildArch=x64 -os=Linux
```

Comment:
Pre-existing, please fix: 'Build Managed' and 'Build Native' should be on same heading level (###) - 'Build Managed' is just at "##". It affects more headings below. Anyway, please make it somehow sensible. Maybe upgrading everything above one level up is the right solution ...